app/components/web-components/light-table/vl-delete-user/component.js

```diff
@@ -1,27 +1,30 @@
 import Component from '@ember/component';
 import { alias } from '@ember/object/computed';
 import { inject } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 
 export default Component.extend({
   user: alias('row'),
   store: inject(),
   isDeleting: false,
+  @tracked isVerifying: false,
```

Comment:
`@tracked` is iets voor Glimmer components denk ik. Dit is geen Glimmer component (`@ember/component` vs `@glimmer/component`). `isVerifying` zou zich hier sowieso `tracked` moeten gedragen :-)

Reply:
Zonder @tracked geeft hij een error op de assignment van isVerifying in de `deleteAccount` action
![image](https://user-images.githubusercontent.com/11557630/79866293-d6eb2580-83dc-11ea-8a0e-7a4e8d128748.png)

Reply:
:thinking: Ze geven 2 opties om dit te resolven. Jij hebt de eerste gekozen (`@tracked`) ... ik had de 2e (`@ember/object#set`) in mn hoofd. Heb niet genoeg achtergrond, maar vind straf dat het werkt, omdat de glimmer component vs  ember component echt andere klassen zijn. Heb ze in ieder geval nog nooit zo gemixed gezien ... 'k weet het niet ...

Reply:
Yes, hoe het mixed werkt.. weet ik ook niet. Ik ging er echter vanuit dat we in de toekomst toch richting `glimmer` components gaan dus leek het me slim om reeds een nieuwe syntax te gebruiken zodat er minder refactoring nodig is in de toekomst toe.